Assets/MixedRealityToolkit-SDK/StandardAssets/Models/GizmoRight.fbx.meta

```diff
@@ -0,0 +1,132 @@
+fileFormatVersion: 2
+guid: 55c254ea7a3a8c8419d144de824249e9
+ModelImporter:
+  serializedVersion: 22
+  fileIDToRecycleName:
+    100000: //RootNode
+    100002: pCone1
+    100004: pCone2
+    100006: pCone3
+    100008: pCone4
+    100010: pCone5
+    100012: pCone6
+    100014: pCylinder1
+    100016: pCylinder2
+    100018: pCylinder3
+    100020: pCylinder4
+    100022: transform1
+    100024: transform2
+    100026: transform3
+    100028: transform4
+    100030: transform5
+    100032: transform6
+    100034: transform7
+    100036: transform8
+    100038: transform9
+    400000: //RootNode
+    400002: pCone1
+    400004: pCone2
+    400006: pCone3
+    400008: pCone4
+    400010: pCone5
+    400012: pCone6
+    400014: pCylinder1
+    400016: pCylinder2
+    400018: pCylinder3
+    400020: pCylinder4
+    400022: transform1
+    400024: transform2
+    400026: transform3
+    400028: transform4
+    400030: transform5
+    400032: transform6
+    400034: transform7
+    400036: transform8
+    400038: transform9
+    2100000: phong2
+    2100002: phong4
+    2100004: phong3
+    2300000: pCone6
+    3300000: pCone6
+    4300000: pCone6
+  externalObjects: {}
+  materials:
+    importMaterials: 1
```

Comment:
let's turn off import materials and use the standard materials we've setup.

Reply:
I can do that